userapi/src/test/java/fk/prof/userapi/verticles/UserapPolicyiListingAPITest.java

```diff
@@ -0,0 +1,158 @@
+                    context.assertEquals(userapiGetAppIdsURI.substring(UserapiApiPathConstants.LIST_POLICY_API_PREFIX.length()), routingContext.normalisedPath()+"?"+routingContext.request().query());
+                    routingContext.response().end(Json.encode(Sets.newSet(APP_ID)));
+                });
+        backendServer.requestHandler(router::accept);
+        backendServer.listen(backendPort, result -> {
+            if (result.succeeded()) {
+                client.getNow(userapiPort, "localhost", userapiGetAppIdsURI, httpClientResponse -> {
+                    context.assertEquals(httpClientResponse.statusCode(), HttpResponseStatus.OK.code());
+                    httpClientResponse.bodyHandler(buffer -> {
+                        context.assertEquals(buffer.toJsonArray().size(), 1);
+                        context.assertTrue(buffer.toString().contains(APP_ID));
+                        async.complete();
+                    });
+                });
+            } else {
+                context.fail();
+            }
+        });
+    }
+
+    @Test
+    public void TestGetClusterIdsRoute(TestContext context) throws Exception {
+        final Async async = context.async();
+        String userapiGetClusterIdsURI = UserapiApiPathConstants.LIST_POLICY_API_PREFIX + "/clusters/" + APP_ID + "?prefix=" + CLUSTER_ID.substring(0, 1 + new Random().nextInt(CLUSTER_ID.length() - 1));
+        Router router = Router.router(vertx);
+        UserapiHttpHelper.attachHandlersToRoute(router, HttpMethod.GET,
+                UserapiApiPathConstants.POLICY_CLUSTERS_FOR_APP.substring(UserapiApiPathConstants.LIST_POLICY_API_PREFIX.length()), routingContext -> {
+                    context.assertEquals(userapiGetClusterIdsURI.substring(UserapiApiPathConstants.LIST_POLICY_API_PREFIX.length()), routingContext.normalisedPath()+"?"+routingContext.request().query());
+                    routingContext.response().end(Json.encode(Sets.newSet(CLUSTER_ID)));
+                });
+        backendServer.requestHandler(router::accept);
+        backendServer.listen(backendPort, result -> {
+            if (result.succeeded()) {
+                client.getNow(userapiPort, "localhost", userapiGetClusterIdsURI, httpClientResponse -> {
+                    context.assertEquals(httpClientResponse.statusCode(), HttpResponseStatus.OK.code());
+                    httpClientResponse.bodyHandler(buffer -> {
+                        context.assertEquals(buffer.toJsonArray().size(), 1);
+                        context.assertTrue(buffer.toString().contains(CLUSTER_ID));
+                        async.complete();
+                    });
+                });
+            } else {
+                context.fail();
+            }
+        });
+    }
+
+
+    @Test
+    public void TestGetProcNamesRoute(TestContext context) throws Exception {
```

Comment:
As mentioned in earlier PRs, method names are not capitalized

Reply:
Made the change [here](https://github.com/Flipkart/fk-prof/commit/97901adac528536e4cae9148452121def9ba62d8)